{"messages":[{"error_id":"Release","references":[{"column":13,"file":"/home/yo30qaqy/data-race-detection-benchmark-suite/results-20250730-140308/CoVer-MUST/static/038-MPI-static-loop-array-bounds-local-no.c","line":42},{"column":13,"file":"/home/yo30qaqy/data-race-detection-benchmark-suite/results-20250730-140308/CoVer-MUST/static/038-MPI-static-loop-array-bounds-local-no.c","line":42}],"text":"Found call to MPI_Get at /home/yo30qaqy/data-race-detection-benchmark-suite/results-20250730-140308/CoVer-MUST/static/038-MPI-static-loop-array-bounds-local-no.c:42:13 which is in conflict with MPI_Get at /home/yo30qaqy/data-race-detection-benchmark-suite/results-20250730-140308/CoVer-MUST/static/038-MPI-static-loop-array-bounds-local-no.c:42:13 before release","type":"Local Data Race - Local write by call"}]}
